lib/plugins/alignedTable.js

```diff
@@ -211,13 +199,11 @@ function table(state, startLine, endLine, silent) {
   tableLines[1] = tbodyLines[1] = nextLine;
   state.line = nextLine;
   return true;
-}
-
+};
 
 module.exports = function aligned_table_plugin(md, options) {
   options = typeof options === 'object' ? options : {};
 
   md.renderer.rules.aligned_table = md.renderer.rules.html_block;
   md.block.ruler.before('table', 'aligned_table', table, {'alt': [ 'paragraph', 'reference' ]});
-
-};
+};
```

Comment:
👌  I was relying on EditorConfig. Which is missing on project. So I will add it.